omod/src/main/java/org/openmrs/module/attachments/web/controller/AttachmentsController.java

```diff
@@ -35,6 +28,13 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.multipart.MultipartHttpServletRequest;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Iterator;
+
+import static org.openmrs.module.attachments.AttachmentsContext.getContentFamily;
+
+@Deprecated
```

Comment:
Can you also add the corresponding Javadoc (see [here](https://docs.oracle.com/javase/1.5.0/docs/guide/javadoc/deprecation/deprecation.html#javadoc_tag))?

When a developer stumbles upon a deprecated API he/she should be pointed to _what_ deprecates the API.

Reply:
is it enough to mention that it's replaced by AttachmentResource, or more informations should be provided ?

Reply:
It's enough, and best it to link to the new place in the code using `{@link}`.